fili-core/src/test/groovy/com/yahoo/bard/webservice/data/QueryBuildingTestingResources.groovy

```diff
@@ -55,50 +59,33 @@ public class QueryBuildingTestingResources {
     public Dimension d6, d7, d8, d9, d10, d11, d12, d13
     public LogicalMetric m1, m2, m3, m4, m5, m6
 
-    public Interval interval1
-    public Interval interval2
-    public Interval interval3
+    public Interval interval1, interval2, interval3
 
-    public PhysicalTable emptyFirst, partialSecond, wholeThird, emptyLast
-    public PhysicalTable t1h, t1d, t1hShort, t2h, t3d, t4h1, t4h2, t4d1, t4d2, t5h
-
-    //Volatility testing
-    public PhysicalTable volatileHourTable = new PhysicalTable("hour", HOUR.buildZonedTimeGrain(UTC), [:])
-    public PhysicalTable volatileDayTable = new PhysicalTable("day", DAY.buildZonedTimeGrain(UTC), [:])
+    public ConcretePhysicalTable t1h, t1d, t1hShort, t2h, t3d, t4h1, t4h2, t4d1, t4d2, t5h, emptyFirst, partialSecond, wholeThird, emptyLast, tna1236d, tna1237d, tna167d, tna267d, volatileHourTable, volatileDayTable
 
     public VolatileIntervalsService volatileIntervalsService
 
-    public TableGroup tg1h, tg1d, tg1Short, tg2h, tg3d, tg4h, tg5h, tg6h, tg1All
-    LogicalTable lt12, lt13, lt14, lt1All
-    LogicalTableDictionary logicalDictionary
-    TableIdentifier ti2h, ti2d, ti3d, ti4d, ti1All
+    public TableGroup tg1h, tg1d, tg1Short, tg2h, tg3d, tg4h, tg5h, tg6h, tg1All, tgna
+
+    public LogicalTable lt12, lt13, lt14, lt1All, ltna
+
+    public LogicalTableDictionary logicalDictionary
 
-    // Tables with non-aggregatable dimensions
-    public PhysicalTable tna1236d, tna1237d, tna167d, tna267d
-    public TableGroup tgna
-    LogicalTable ltna
-    TableIdentifier tina
+    public TableIdentifier ti2h, ti2d, ti3d, ti4d, ti1All, tina
 
-    TemplateDruidQuery simpleTemplateQuery
-    TemplateDruidQuery simpleNestedTemplateQuery
-    TemplateDruidQuery complexTemplateQuery
-    TemplateDruidQuery simpleTemplateWithGrainQuery
-    TemplateDruidQuery complexTemplateWithInnerGrainQuery
-    TemplateDruidQuery complexTemplateWithDoubleGrainQuery
+    public TemplateDruidQuery simpleTemplateQuery, simpleNestedTemplateQuery, complexTemplateQuery, simpleTemplateWithGrainQuery, complexTemplateWithInnerGrainQuery, complexTemplateWithDoubleGrainQuery
```

Comment:
Doing this actually violates the general style guide we follow: https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations